drivers/usb_c/tcpc/ps8xxx.c

```diff
@@ -718,8 +628,7 @@ static int ps8xxx_dev_init(const struct device *dev)
 	}
 
 	k_work_init_delayable(&data->init_dwork, ps8xxx_init_work_cb);
-	k_work_schedule_for_queue(&k_sys_work_q, &data->init_dwork,
-				  K_MSEC(CONFIG_USBC_TCPC_PS8XXX_INIT_DELAY));
+	k_work_schedule(&data->init_dwork, K_MSEC(CONFIG_USBC_TCPC_PS8XXX_INIT_DELAY));
```

Comment:
Non-blocking comment.  This change isn't related to using the common tcpci functions.